docs/src/main/sphinx/connector/memory.md

```diff
@@ -101,6 +101,3 @@ in the catalog file.
   or may return partial data.
 - When the coordinator fails/restarts, all metadata about tables is
   lost. The tables remain on the workers, but become inaccessible.
-- This connector does not work properly with multiple
```

Comment:
> Correct memory connector docs

I would recommend clarifying what was corrected. The current commit title is too general.

Reply:
Done